docs/examples.md

```diff
@@ -1,45 +1,48 @@
 # Examples
 
-This Example has been taken with a Ledger nano. 
-The OS in this case OSX (17.7.0 Darwin Kernel Version 17.7.0). In Linux the paths should be shorter.
-
+This Example has been taken with a Ledger nano S.
 
 ```
 ./hwi.py enumerate
-[{"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@0/IOUSBHostHIDDevice@14200000,0", "serial_number": "0001"}, {"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@1/IOUSBHostHIDDevice@14200000,1", "serial_number": "0001"}]```
+[{"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@0/IOUSBHostHIDDevice@14200000,0", "serial_number": "0001"}, {"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@1/IOUSBHostHIDDevice@14200000,1", "serial_number": "0001"}]
 ```
+The OS in this case is OSX (17.7.0 Darwin Kernel Version 17.7.0). In Linux the 
+"path" is shorter.
 
-To extract the Extended Master Public  
+## Useful xpubs to extract
 
-```
- ./hwi.py -t "ledger" -d "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@0/IOUSBHostHIDDevice@14200000,0" getmasterxpub
-=> b'e0c4000000'
-<= b'1b30010208010003'9000
-=> b'f026000000'
-<= b''6d00
-=> b'e04000000d038000002c8000000080000000'
-<= b'4104f4b866b49fb76529a076a1c5b25216c1f4b970cb8e3db9874beb15c5371fdb93747fde522d63be4a564dcda8a71c889f5165eac2990cafee9d416141ae8b09c722313667774c7a76697157783146317a653365676850464d58655438666a57466f4b66f9a82310c4530360ec3fee42049fbb7a3c0bfa72fdf2c5b25b09f1c3df21c938'9000
-=> b'e040000009028000002c80000000'
-<= b'4104280c846650d7771396a679a55b30c558501f0b5554160c1fbd1d7301c845dacc10c256af2c8d6a13ae4a83763fa747c0d4c09cfa60bfc16714e10b0a938a4a6a2231485451557a6535486571334872553755435174564652745a535839615352674a65d62f97789c088a0b0c3ed57754f75273c6696c0d7812c702ca4f2f72c8631c04'9000
-{"xpub": "xpub6CyidiQae2HF71YigFJqteLsRi9D1EvZJm1Lr4DWWxFVruf3vDSbfyxD9znqVkUTUzc4EdgxDRoHXn64gMbFXQGKXg5nPNfvyVcpuPNn92n"}
+Starting from version 0.17, It is possible to to retrieve the Unspent 
```

Comment:
nit: You should specify what `0.17` refers to. I assume you mean Bitcoin Core, so it should say "Bitcoin Core 0.17"

Reply:
Done and also corrected a typo and restricted the lines to 80 characters.
I have tried to squash but it seems there's other commits in between.
```
pick 6589a95 Errors corrected, repetitions reduced
pick 5c29bed Extracting xpubs from hwww and using them with Bitcoin Core
pick e5d302b Add debug option and setup a debug logger
pick 5bb49e5 Only have ledger debug statements when debug is enabled
pick 7bfd11f Turn debug prints into logging.debug
pick 48228bd Proper return values and typo fixes for Digital Bitbox
pick 21e4122 Added "Bitcoin Core", formatting and typos corrected.
```